climada/hazard/tc_tracks.py

```diff
@@ -1016,10 +1016,10 @@ def to_geodataframe(self, as_points=False):
         """Transform this TCTracks instance into a GeoDataFrame.
 
         Parameters:
-            as_points (bool): If true, construct LineString or single Point
-                geometries. A point is created if a track has length one.
-                If set to false, the geometries are returned as points with an
-                additional timestamp column.
+            as_points : bool, optional
+                If False (default), one row per track with a LineString as geometry (or Point
+                geometry for tracks of length one) and all track attributes as dataframe columns.
+                If True, one row per track point, with variable values in addition to attributes.
```

Comment:
Could you clarify what 'variable values' are? Which variables? Also, what attributes? Do you mean it in the pythonic sense?

Reply:
I meant in the xarray sense. Attributes are unique for an xarray object i.e. for a track, e.g. track id, name, category, etc. Variables are for each time step, e.g. max sustained wind, central pressure, etc.
Hence, attributes can be set for whole tracks or to individual points (same for all points within a given track), while variables cannot be given to the line, but only to the points.

Reply:
Ok. This should be explained better in the docstring. Currently it is very confusing.

Reply:
Ok I've improved it, better now?

Reply:
Excellent!